frontend/src/views/Home.js

```diff
@@ -0,0 +1,137 @@
+            { id: "2", name: "Rock cycle", description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc maximus, nulla ut commodo sagittis, sapien dui mattis dui, non pulvinar lorem felis nec erat", img: "https://bulma.io/images/placeholders/256x256.png" },
+            { id: "3", name: "Rock cycle", description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc maximus, nulla ut commodo sagittis, sapien dui mattis dui, non pulvinar lorem felis nec erat", img: "https://bulma.io/images/placeholders/256x256.png" },
+        ],
+        gyms: [
+            { id: "1", name: "Octava compania", img: "https://bulma.io/images/placeholders/256x256.png" },
+            { id: "2", name: "Octava compania", img: "https://bulma.io/images/placeholders/256x256.png" },
+        ],
+        recommender: { name: "Cristina", description: "Al fin voy al gimnasio, gracias a mi amiga", img: "https://bulma.io/images/placeholders/256x256.png" }
+    }
+}
+
+function Home() {
+
+    const [homeData, setHomeData] = useState({
+        activities: [],
+        gyms: [],
+        recommender: [],
+    })
+    useEffect(() => {
+        const fetchData = async () => {
+            const data = fetchDataFromApi()
+            setHomeData(data)
+        }
+        fetchData()
+    }, [])
+    
+    return (
+        <div id="landing-content">
+            <div className="container">
+                <div className="columns">
+                    <div className="column">
+                        <h1 className="title is-size-1 has-text-weight-bold" style={{paddingBottom: 100}}>
+                        Entrena con quien <br></br>quieras y donde sea
+                        </h1>
+                        <ul>
+                            <li className="level-left is-size-5">
+                                <span className="icon is-large has-text-green">
+                                    <i className="far fa-hand-point-right fa-2x"></i>
+                                </span>
+                                <span className="subtitle" style={{paddingLeft: 10}}>Con tu amigo, profe o compañero</span>
+                            </li>
+                            <li className="level-left is-size-5">
+                                <span className="icon is-large has-text-green">
+                                    <i className="far fa-hand-point-right fa-2x"></i>
+                                </span>
+                                <span className="subtitle" style={{paddingLeft: 10}}>Enterate de nuevas clases a <br></br> precios increíbles, y hasta <br></br> gratis.</span>
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="column">
+                        <Recommender name={homeData.recommender.name} img={homeData.recommender.img} description={homeData.recommender.description} />
```

Comment:
nit: I think you should be able to use something like:
```js
<Recommender {...homeData.recommender}>
```

the spread operator will end up passing all fields in `homeData.recommender` as props to Recommender component

whenever you see a `nit` comment of mine, interpret it as: feel free to decide whether to do this change or not, it's just a suggestion